README.md

```diff
@@ -37,10 +39,7 @@ Select 1:
 
 Describe the rules of scope in JavaScript.
 
-Your Answer:
-```text
-```
-
+Your Answer: Scope deals with the terms local and global. When a variable has a global scope, it is defined outside of functions or expressions, making it accessible use and manipulate by all functions, loops, or expressions. In contrast, local variables have a smaller scope and are initiatiated inside a function.
 
```

Comment:
Pretty much. Worth noting that local variables defined inside a function can be accessed by functions of a higher scope.